### PR TITLE
[PORT] tgui antag info popup

### PR DESCRIPTION
**Pull request by Absolucy**
## About The Pull Request

This partially ports https://github.com/tgstation/tgstation/pull/60311 (only the antag ui stuff, not the flavor stuff)

### Popups Implemented
- [x] Traitor
- [x] Incursion
- [x] Blood Brother
- [x] Brainwashed
- [x] Blob
- [x] Changeling
- [x] Heretic (done before this PR)
- [x] Malf AI
- [x] Nightmare
- [x] Wizard
- [x] Abductor Agent
- [x] Abductor Scientist
- [x] Generic/Custom Antag
- [x] Blood Cultist
- [x] Nuclear Operative
- [ ] Revenant
- [x] Spider
- [ ] Xenomorph
- [ ] Space Dragon

***I need help writing mini-tutorials / flavor-text for the remaining antags!!***

## Why It's Good For The Game

Helps players out more, framework for future traitor backstory stuff and memory reworks and whatnot.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

</details>

## Changelog
:cl: Absolucy, tralezab
add: Added tgui-based antag info popups for multiple antags, replacing the previous basic popups and info.
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
I always thought this looked good to orient new players! I feel like that flavour text could be used and tweaked on stuff we want for the server(for example there iirc they have different traitor factions like Donk Co., Gorlex Marauders, etc and suggest them who they're allies and enemie) so we would need to discuss if we want something similar to orient players on certain stuff we want and expect from an antag

Good port!

File: code/modules/antagonists/changeling/changeling.dm [hunk 1]
```diff
@@ -396,7 +404,6 @@
 	to_chat(owner.current, "<b>You must complete the following tasks:</b>")
 	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ling_aler.ogg', 100, FALSE, pressure_affected = FALSE, use_reverb = FALSE)
 
-	owner.announce_objectives()
```
Comment (itsmeow) [inline code/modules/antagonists/changeling/changeling.dm:399]: why

File: code/modules/antagonists/traitor/datum_traitor.dm [hunk 2]
```diff
@@ -206,10 +225,10 @@
 			.=2
 
 /datum/antagonist/traitor/greet()
-	to_chat(owner.current, "<span class='alertsyndie'>You are the [owner.special_role].</span>")
-	owner.announce_objectives()
-	if(should_give_codewords)
-		give_codewords()
+	var/list/msg = list()
+	msg += "<span class='alertsyndie'>You are the [owner.special_role].</span>"
+	msg += "<span class='alertsyndie'>Use the 'Open [owner.special_role] Information' action at the top left in order to re-review your objectives, codewords, and such!</span>"
```
Comment (itsmeow) [inline code/modules/antagonists/traitor/datum_traitor.dm]:
```suggestion
	msg += "<span class='alertsyndie'>Use the 'Open [owner.special_role] Information' action at the top left in order to review your objectives and codewords!</span>"
```

File: tgui/packages/tgui/interfaces/AntagInfoIncursion.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      </Stack>
+    </Section>
+  );
+};
+
+const BasicLoreSubsection = (_props, _context) => {
+  return (
+    <Section>
+      <BlockQuote>
+        You have formed a team of Syndicate members with a similar mindset and must infiltrate the ranks of the station!
```
Comment (itsmeow) [inline tgui/packages/tgui/interfaces/AntagInfoIncursion.tsx]:
```suggestion
        You have joined a team of Syndicate agents with shared goals and must infiltrate the ranks of the station!
```

File: tgui/packages/tgui/interfaces/AntagInfoNukeOp.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+};
+
+const MissionNormal = (_props, context) => {
+  const { data } = useBackend<Info>(context);
+  return (
+    <BlockQuote>
+      <p>
+        Your mission is simple, blow up the station. To achieve this your team must get the Nuclear Authentification Disk. Once
+        you have the disk, it&apos;s only a matter of getting the nuke aboard the station, arming it and then getting out before
+        it pulverizes you in it&apos;s blast.
```
Comment (itsmeow) [inline tgui/packages/tgui/interfaces/AntagInfoNukeOp.tsx]:
```suggestion
        it pulverizes you in its blast.
```

File: tgui/packages/tgui/interfaces/AntagInfoNukeOp.tsx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  );
+};
+
+const MissionNormal = (_props, context) => {
+  const { data } = useBackend<Info>(context);
+  return (
+    <BlockQuote>
+      <p>
+        Your mission is simple, blow up the station. To achieve this your team must get the Nuclear Authentification Disk. Once
+        you have the disk, it&apos;s only a matter of getting the nuke aboard the station, arming it and then getting out before
```
Comment (itsmeow) [inline tgui/packages/tgui/interfaces/AntagInfoNukeOp.tsx]:
```suggestion
        you have the disk, it&apos;s only a matter of getting the nuke aboard the station, arming it, starting the timer, and then getting out before
```

File: tgui/packages/tgui/interfaces/AntagInfoNukeOp.tsx [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+        it pulverizes you in it&apos;s blast.
+      </p>
+      {(!!data.leader && (
+        <p>
+          You have been made the leader of this mission, you must coodrinate with your team and devise a strategy. If you are
```
Comment (itsmeow) [inline tgui/packages/tgui/interfaces/AntagInfoNukeOp.tsx]:
```suggestion
          You have been made the leader of this mission, you must coordinate with your team and devise a strategy. If you are
```

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Comment by itsmeow** [comment]
*None* of the UIs should use the default NT theme, use generic if there is no relevant theme

File: code/modules/antagonists/changeling/changeling.dm (hunk 1 above)
Comment (Absolucy) [reply code/modules/antagonists/changeling/changeling.dm:399]: they show up on the popup anyways, but eh, i can add it back

**Comment by Absolucy** [comment]
@itsmeow they aren't, afaik?
`neutral` is _NOT_ the NT theme, it's the one with the incognito guy in the back, which imo fits well with antags
image
(AntagInfoGeneric for example)

**Comment by itsmeow** [comment]
I was just peeking at the screenshots, outdated so if that's true then it's all good

**Comment by Absolucy** [comment]
Ah yeah, some are outdated.

File: code/modules/antagonists/changeling/changeling.dm (hunk 1 above)
Comment (itsmeow) [reply code/modules/antagonists/changeling/changeling.dm:399]: yea

**Review by itsmeow — APPROVED** [review]
very cool and epic

**Pull request merged**